Source/Core/Editor/EditorUtils.cs

```diff
@@ -2,17 +2,16 @@
 // Licensed under the Apache License, Version 2.0
 // Modifications copyright (c) 2021-2023 MindPort GmbH
 
+using JetBrains.Annotations;
```

Comment:
I suggest removing JetBrains.Annotations this and [PublicAPI] as we don't use JetBrains.
This is the only file I found using JetBrains.

Reply:
Ok